sdk/samples/router/general.py

```diff
@@ -50,6 +50,11 @@ def main(turing_api: str, project: str):
             id='ring-ding-ding',
             endpoint='http://fox-says.ring-ding-ding',
             timeout='20ms'
+        ),
```

Comment:
Thanks for updating the sample, I missed it.

Extreme nit: Can the route id be anything but "nothing"? 😆  When used like below, it seems like "nothing" has some significance other than just being some route name.

```python
default_route_id="nothing",
```

Reply:
Okay I'll put `control` as the name of the default route name instead 😅